Kong/docs.konghq.com
# [DOCU-3304][DOCU-3305] Fonts and sidebar

### Description

Very in progress, don't merge without a bunch more work.

Updating the docs site font to Roboto, Kong's "official" font.
This font also provides a lot more contrast than our current. See comparison:

<img width="400" alt="Screenshot 2023-06-30 at 3 02 00 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/3f1b4f7f-c2bd-42a7-a111-a7810e1011f1">
<img width="400" alt="Screenshot 2023-06-30 at 2 55 55 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/f29a1fdc-dceb-4853-a0d6-53ae0a9c7dce">

Also started messing with the left sidebar, but needs a lot more work to look right + spacing adjustments.

https://konghq.atlassian.net/browse/DOCU-3304
https://konghq.atlassian.net/browse/DOCU-3305

### Testing instructions

Reviewers, I need some opinions!

Currently, in the PR, the font size is 16. Should it be smaller, down to 15? here's what that comparison looks like:
<img width="400" alt="Screenshot 2023-06-30 at 2 55 55 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/32cba8aa-3a37-4ced-a92b-82580cb69fdd">
<img width="400" alt="Screenshot 2023-06-30 at 2 55 26 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/fb43a15c-4551-4387-ac2d-d61976a20d19">

Please check lots of different types of pages to see where else fonts and spacing need adjusting. So far, the to do list includes:
* Adjust padding & margins in left sidenav
* Adjust whitespace in ordered lists
* Do the carets work, or do they need to be smaller? 

### Checklist
